documentation/deploying/master.adoc

```diff
@@ -20,3 +20,5 @@ include::assemblies/deploying/assembly-deploy-verify.adoc[leveloffset=+1]
 include::assemblies/metrics/assembly-metrics.adoc[leveloffset=+1]
 //Upgrading the deployment
 include::assemblies/upgrading/assembly-upgrade.adoc[leveloffset=+1]
+//Downgrading the deployment
+include::modules/upgrading/proc-downgrade-cluster-operator.adoc[leveloffset=+1]
```

Comment:
Just checking that the correct module is included in the `master.adoc`?

Reply:
For the time being, yes. There's a new module for downgrading, but we're concentrating only on the CO downgrade for the time being. Other files exists, but won't be included yet.